.evergreen/run-mongodb-aws-ecs-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Don't trace since the URI contains a password that shouldn't show up in the logs
```

Comment:
So I know you probably copied this from the Java Driver but the URI doesn't actually contain a password for ECS test. @jyemin I can change this line in the same script in the Java driver to enable xtrace.